src/executables/util_sa.c

```diff
@@ -1575,7 +1575,9 @@ diagdb (UTIL_FUNCTION_ARG * arg)
 
   if (class_name && class_list_file)
     {
-      goto print_diag_usage;
+      errno = EINVAL;
+      perror ("The -n and -i options cannot be used together. ");
```

Comment:
perror가 errno 에러 메시지를 포함해서 출력한다는걸 잊었네요. stderr로 출력해주면 충분할 것 같습니다.
```suggestion
      fprintf (stderr, "The -n and -i options cannot be used together.");
```

Reply:
제안해주신 코드로는 화면에 에러 메시지가 출력되지 않아서 아래와 같이 '\n'를 추가하였습니다. 이렇게 진행해도 될까요?
`fprintf (stderr, "The -n and -i options cannot be used together.\n");

Reply:
넵. 이런건 안물어 보셔도 될 것 같아요. :D